src/sealed.rs

```diff
@@ -46,3 +49,7 @@ impl<T> Sealed for AddTypeToModule<T> {}
 impl<T> Sealed for AddClassToModule<T> {}
 impl Sealed for PyMethodDef {}
 impl Sealed for ModuleDef {}
+
+impl<T: PyObjectInit<T>> Sealed for T {}
```

Comment:
```suggestion
```
I don't think we should add a blanket implementation for `Sealed`. If we implement it for types that we don't control we have to be careful to not open the seal. In this case it seems we don't need that impl at all, so we should just remove it.

Reply:
Thanks for catching that, I have no idea why I added that. Me just being silly I guess